database/multistmt/parse.go

```diff
@@ -58,21 +44,71 @@ func Parse(reader io.Reader, delimiter []byte, maxMigrationSize int, replacement
 	// completed statements, contents of accum will be dumped in here
 	stmts := make([][]byte, 0, 1000)
 
+	tmp := make([]byte, 0, 10)
```

Comment:
Why do we initialize `tmp`? It's initialized inside the loop already.

```suggestion
	var tmp []byte
```

Reply:
Yes, good catch, ty